src/test/java/ua/od/deadlinks/CheckDeadLinksTest.java

```diff
@@ -0,0 +1,67 @@
+package ua.od.deadlinks;
+
+import org.junit.jupiter.api.*;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+public class CheckDeadLinksTest {
+
+    private final ByteArrayOutputStream outContent = new ByteArrayOutputStream();
+    private final ByteArrayOutputStream errContent = new ByteArrayOutputStream();
+    private final PrintStream originalOut = System.out;
+    private final PrintStream originalErr = System.err;
+
+
+    private final String url = "https://github.com";
+    private final String expected = "{\n" +
+            "  \"url\": \"https://github.com\",\n" +
+            "  \"404\": {\n" +
+            "    \"size\": 3,\n" +
+            "    \"urls\": [\n" +
+            "      \"\",\n" +
```

Comment:
@alexl-2018 why this is empty?

Reply:
Because there are the inner html code or javascript.